rosbag2_storage_sqlite3/src/rosbag2_storage_sqlite3/sqlite_storage.cpp

```diff
@@ -525,9 +525,11 @@ void SqliteStorage::fill_topics_and_types()
     "SELECT name, type, serialization_format FROM topics ORDER BY id;");
   auto query_results = statement->execute_query<std::string, std::string, std::string>();
 
+  std::string type_description_hash = "";  // TODO(james-rms) populate type description hash
```

Comment:
Are these TODOs to be addressed before merging this PR? or are they for a followup

Reply:
@james-rms could you please add new field `type_description_hash` to the `topic` table to facilitate TODOs in `sqlite_storage.cpp` file?

To have backward compatibility will need to increase `schema_version` in `schema` table and add 
check for `if (database_->field_exists("topics", "type_description_hash"))` where need to use a new sql statement with type_hash.